src/rendering/src/components/NonSitecore/SummitLayout.tsx

```diff
@@ -0,0 +1,615 @@
+                      id: '{1F4B781B-F2A5-5647-99DF-C0C369162C4D}',
+                      querystring: '',
+                      anchor: '',
+                      target: '',
+                      title: '',
+                      linktype: 'internal',
+                      text: '',
+                      url: '/PLAY/playwebsite/home/speakers',
+                      href: '/speakers',
+                    },
+                  },
+                },
+              },
+              {
+                displayName: 'Vendors',
+                field: {
+                  jsonValue: {
+                    value: {
+                      class: '',
+                      id: '{774E44E8-0F30-5879-B847-AD233FFB41AA}',
+                      querystring: '',
+                      anchor: '',
+                      target: '',
+                      title: '',
+                      linktype: 'internal',
+                      text: '',
+                      url: '/PLAY/playwebsite/home/vendors',
+                      href: '/vendors',
+                    },
+                  },
+                },
+              },
+              {
+                displayName: 'Sponsors',
+                field: {
+                  jsonValue: {
+                    value: {
+                      class: '',
+                      id: '{66C99E47-7BBF-52D1-B1D7-4662B850744A}',
+                      querystring: '',
+                      anchor: '',
+                      target: '',
+                      title: '',
+                      linktype: 'internal',
+                      text: '',
+                      url: '/PLAY/playwebsite/home/sponsors',
+                      href: '/sponsors',
+                    },
+                  },
+                },
+              },
+              {
+                displayName: 'About Us',
+                field: {
+                  jsonValue: {
+                    value: {
+                      class: '',
+                      id: '{2717574C-48A0-5469-85A8-A332DF71F1E4}',
+                      querystring: '',
+                      anchor: '',
+                      target: '',
+                      title: '',
+                      linktype: 'internal',
+                      text: '',
+                      url: '/PLAY/playwebsite/home/about-us',
+                      href: '/about-us',
+                    },
+                  },
+                },
+              },
+            ],
+          },
+        },
+      },
+    },
+    params: {
+      CacheClearingBehavior: 'Clear on publish',
+    },
+  } as unknown as MainNavigationProps;
+
+  return (
+    <>
+      <Head>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <header>
+        <HeaderContent {...headerContentProps} />
+        <MainNavigation {...mainNavigationProps} />
+      </header>
+      <main>
+        <HeaderCdpMessageBar />
+        <div className="summit-main-container">{props.children}</div>
```

Comment:
I am not sure it's necessary to have it al all, I just changed it because it said "shop-main-container" before (it was copied from the ShopLayout I suppose) and it was carrying over unwanted styles.